app/views/admin/articles/_article_item.html.erb

```diff
@@ -0,0 +1,191 @@
+<% decorated_article = article.decorate %>
+
+<article class="js-individual-article crayons-card p-6 flex flex-col gap-4"
+     data-controller="article"
+     data-article-id-value="<%= article.id %>"
+     data-article-pin-path-value="<%= stories_feed_pinned_article_path %>"
+     data-article-bg-highlighted-class="bg-highlighted"
+     data-article-border-highlighted-class="border-highlighted"
+     data-action="ajax:success@document->article#ajaxSuccess">
+
+  <header>
+    <% if decorated_article.pinned? || article.featured || article.approved || (!article.published? && article.published_from_feed?) || article.video || article.user&.warned? %>
+      <div class="flex gap-1 mb-1">
+        <% if !article.published? && article.published_from_feed? %>
+          <span class="c-indicator c-indicator--danger">Unpublished</span>
+        <% end %>
+        <% if decorated_article.pinned? %>
+          <span class="c-indicator c-indicator--warning" data-testid="pinned-indicator">Pinned</span>
```

Comment:
Do you think we should add these and some others to the i81n file?

Reply:
Applied this change. Shifted strings to yml files.